src/librustdoc/html/render/mod.rs

```diff
@@ -1577,7 +1579,7 @@ impl Context {
         }
     }
 
-    /// Construct a map of items shown in the sidebar to a plain-text summary of their docs.
+    /// Construct a map of items shown in the sidebar to a summary of their docs.
```

Comment:
This is still rendering as plain-text, right? Why did you change the comment?

Reply:
Thanks! I think I forgot to update it when I switched back to using plaintext for alt-text.